app/mailer_modules/send_email.js

```diff
@@ -0,0 +1,58 @@
+const nodemailer = require('nodemailer');
+const ejs = require('ejs');
+require('dotenv').config();
+
+const transporter = nodemailer.createTransport({
+    host: 'smtp.gmail.com',
+    port: 465,
+    secure: true,
+    auth: {
+        user: process.env.SENDER_EMAIL, 
+        pass: process.env.SENDER_PASSWORD, 
+    },
+});
+
+/**
+ * Sends email reminders to attendees given information about the yoga class
+ * and an array of the attendees' information. 
+ * 
+ * @param {Object} classInfo Information about the class.
+ * @param {string} classInfo.className Name of the Yoga class.
+ * @param {string} classInfo.startTime Time that the class starts.
+ * 
+ * @param {{firstName : string, lastName : string, email : string}[]} clients Information about the client.
+ */
+async function sendEmails(classInfo, clients){
+    clients.map(async function(client){
+        await ejs.renderFile("../views/email_template.ejs",{
+            firstName : client.firstName,
+            lastName : client.lastName,
+            className : classInfo.className,
```

Comment:
you can just put the entire classInfo in, and they can access these values by doing `classInfo.className` etc in the ejs file.